### PR TITLE
Adding INFN T1 and caching

### PR DESCRIPTION
**Pull request by lmorganti**

### CODE REVIEW

File: topology/INFN-T1/INFN-T1/INFN-T1CachingInfrastructure.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+  T1_STASHCACHE_CACHE:
+    FQDN: xcache.cr.cnaf.infn.it
+    Service: XRootD cache server
+      Description: StashCache cache server 
+    AllowedVOs:
+      - ANY_PUBLIC
+    DN: DC=org, DC=terena, DC=tcs, C=IT, L=Frascati, O=Istituto Nazionale di Fisica Nucleare, CN=ds-102-11-18.cr.cnaf.infn.it
```
Comment (brianhlin) [inline topology/INFN-T1/INFN-T1/INFN-T1CachingInfrastructure.yaml]: Hi, we use the older openssl DN formatting:

```suggestion
    DN: /DC=org/DC=terena/DC=tcs/C=IT/L=Frascati/O=Istituto Nazionale di Fisica Nucleare/CN=ds-102-11-18.cr.cnaf.infn.it
```

**Comment by dcesini** [comment]
Hi, these are the Terena CA released by EGI and installed at cnaf:
rpm -qa | grep ca | grep -i terena
ca_TERENAeSciencePersonalCA3-1.101-1.noarch
subject= /C=NL/ST=Noord-Holland/L=Amsterdam/O=TERENA/CN=TERENA eScience Personal CA 3
ca_TERENA-eScience-SSL-CA-3-1.101-1.noarch
subject= /C=NL/ST=Noord-Holland/L=Amsterdam/O=TERENA/CN=TERENA eScience SSL CA 3

Kind Regards,
   Daniele.

File: topology/INFN-T1/INFN-T1/INFN-T1CachingInfrastructure.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+  T1_STASHCACHE_CACHE:
+    FQDN: xcache.cr.cnaf.infn.it
```
Comment (brianhlin) [inline topology/INFN-T1/INFN-T1/INFN-T1CachingInfrastructure.yaml]:
```suggestion
    FQDN: xcache.cr.cnaf.infn.it
    ID: 1018
```

**Comment by brianhlin** [comment]
@dcesini if you accept both my suggestions (https://github.com/opensciencegrid/topology/pull/738/files), this should be good to go

**Comment by efajardo** [comment]
@brianhlin I am not allowed to commit these suggestions

**Comment by matyasselmeci** [comment]
Looks like @lmorganti has to do it, and also update to the latest master.

**Comment by lmorganti** [comment]
Sorry everybody for being so late: I have accepted both suggestions.
Cheers,
lucia
-----------------------------------------------------------------------------
Dr. Lucia Morganti
@ INFN - CNAF (www.cnaf.infn.it)
Address:   Viale Berti Pichat 6/2, 40127 Bologna (IT)
-----------------------------------------------------------------------------

**Comment by matyasselmeci** [comment]
Oh, I didn't realize this was a new facility and site. In that case, some more work is required:
- Create `topology/INFN-T1/FACILITY.yaml` with the contents:
```
ID: 10094
```
- Copy `template-SITE.yaml` to `topology/INFN-T1/INFN-T1/SITE.yaml` and fill in the values. Use `10256` for the ID.

**Comment by lmorganti** [comment]
Ok, I should have done this

**Comment by matyasselmeci** [comment]
@lmorganti, the resource also needs at least one administrative contact and at least one security contact. I will take care of entering that data, but should I use you for those or someone else?

**Comment by lmorganti** [comment]
Hi, you can use me but please also
vladimir.sapunenko@cnaf.infn.it
enrico.fattibene@cnaf.infn.it
daniele.cesini@cnaf.infn.it
Thank you,
lucia
-----------------------------------------------------------------------------
Dr. Lucia Morganti
@ INFN - CNAF (www.cnaf.infn.it)
Address:   Viale Berti Pichat 6/2, 40127 Bologna (IT)
-----------------------------------------------------------------------------

**Comment by matyasselmeci** [comment]
vladimir.sapunenko@cnaf.infn.it is not registered in our contacts DB, plus the system only supports 3 contacts of each type, so for now I just added you, Enrico, and Daniele. We should get Vladimir registered - I just need a brief profile, e.g. "Site administrator at INFN Tier 1". Afterward, if you want me to swap someone out for him, I can do that too.

**Pull request merged**